### PR TITLE
API: services .json

### PR DESCRIPTION
**Pull request by Arthi-chaud**

### CODE REVIEW

**Comment by AnonymusRaccoon** [comment]
How should the front display a name to the user and how should translations be handled?

File: api/services/youtube.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+					"name": "TITLE",
+					"description": "Title of the uploaded video"
+				},
+				{
+					"name": "DESCRIPTION",
+					"description": "Description of the uploaded video"
+				},
+				{
+					"name": "LIKED_AT",
+					"description": "Date of the video like"
```
Comment (Octopus773) [inline api/services/youtube.json:77]: wth is this on `OnYtPlaylistAdd`, being on a playlist doesn't mean the video is liked

File: api/services/youtube.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+				},
+				{
+					"name": "CHANNEL_TITLE",
+					"description": "Name of the channel the video was uploaded to"
```
Comment (Octopus773) [inline api/services/youtube.json:81]:
```suggestion
					"description": "Name of the channel the video is belonging to"
```

File: api/services/youtube.json [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+			]
+		},
+		{
+			"name": "YtComment",
+			"description": "Comment a Youtube video",
+			"params": [
+				{
+					"name": "videoId",
+					"type": "string",
+					"description": "ID of the video to like"
```
Comment (Octopus773) [inline api/services/youtube.json:111]:
```suggestion
					"description": "ID of the video to comment"
```

**Review by Octopus773 — CHANGES_REQUESTED** [review]

**Review by Octopus773 — APPROVED** [review]

**Review by GitBluub — APPROVED** [review]

**Pull request merged**